lib/stream/fake.go

```diff
@@ -28,10 +29,11 @@ type Fake struct {
 }
 
 // Produce a fake stream. Unlike a real stream, this does not call a subprocess
-// --- instead it just reads from a predefined stream (Bytes).
-func (producer *Fake) Produce() (io.Reader, error) {
+// --- instead it just provides a predefined stream (Bytes).
```

Comment:
tin nit again: on stdout only

Reply:
Thank you so much for catching stuff like this. Fixed in newest version.